src/typeahead-search/typeahead-search.component.ts

```diff
@@ -0,0 +1,125 @@
+import { Search } from "carbon-components-angular/search";
+import { DropdownService } from "carbon-components-angular/dropdown";
+import { I18n } from "carbon-components-angular/i18n";
+import {
+	Component,
+	AfterViewInit,
+	ElementRef,
+	ContentChild,
+	HostListener,
+	OnDestroy
+} from "@angular/core";
+import { NG_VALUE_ACCESSOR } from "@angular/forms";
+import { TypeaheadList } from "./typeahead-list.component";
+
+@Component({
+	selector: "ibm-typeahead-search",
+	templateUrl: "../search/search.component.html",
```

Comment:
Hmmm ... I would have expected `typeahead-search` to embed the existing search instead of extending it ... This is a bit of a messy inheritance situation - we're putting a a lot of responsibility in the parent classes template.